tests/chainerx_tests/unit_tests/routines_tests/test_activation.py

```diff
@@ -75,6 +75,56 @@ def forward_xp(self, inputs, xp):
 ]
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    # Special shapes
+    chainer.testing.product({
+        'shape': [(), (0,), (1,), (2, 0, 3), (1, 1, 1), (2, 3)],
+        'in_dtypes,out_dtype': _in_out_dtypes_math_functions,
+        'input': [-2, 2],
+        'contiguous': [None, 'C'],
+    })
+    # Special values
+    + chainer.testing.product({
+        'shape': [(2, 3)],
+        'in_dtypes,out_dtype': _in_out_float_dtypes_math_functions,
+        'input': [0, float('inf'), -float('inf'), float('nan')],
+        'skip_backward_test': [True],
+        'skip_double_backward_test': [True],
+    })
+))
+class TestLeakyRelu(UnaryMathTestBase, op_utils.ChainerOpTest):
```

Comment:
How about implementing this test using `op_utils.NumpyOpTest` similar to the ReLU test? That should probably get rid of the current Travis dtype failures.